doc/config.toml.tmpl

```diff
@@ -0,0 +1,32 @@
+baseURL = "http://localhost/"
+languageCode = "en-us"
+title = "The Things Network Stack for LoRaWAN"
```

Comment:
Let's explicitly call this documentation for SEO, unless we make this a general Stack landing page